src/app/views/query-runner/query-input/auto-complete/AutoComplete.tsx

```diff
@@ -57,13 +60,34 @@ class AutoComplete extends Component<IAutoCompleteProps, IAutoCompleteState> {
     this.props.contentChanged(userInput);
   };
 
+  public isOverflowing = (input: string) => {
+
+    function getTextWidth(text: string) {
+      const canvas = document.createElement('canvas');
+      const context = canvas.getContext('2d');
+    
+      if (context === null) {
+        return 0;
+      }
+
+      context.font = getComputedStyle(document.body).font;
+      return context.measureText(text).width + 5;
+    }
+
+    return this.element !== null
+      && this.element !== undefined
```

Comment:
```suggestion
    return !!this.element
```
Like such? @omersayshi ?

Thank you for the heads up 👀 Interesting stuff!